### ISSUE TITLE
Attempting to caption images using BLIP Captioning results in a ValueError

### ISSUE THREAD

**Issue by Starbaby8** [bmaltais/kohya_ss#1037]
Normally the code will run an automatic installation for the Blip Captioning Model if not already downloaded, then locate the images and begin captioning, but mine didn't in this instance. Instead I get this: 

```
21:32:19-480227 INFO     Captioning files in C:/Users/nicol/Documents/Taocah_Training/Images/taocah_training_images...
21:32:19-482316 INFO     ./venv/Scripts/python.exe "finetune/make_captions.py" --batch_size="1" --num_beams="1"
                         --top_p="0.9" --max_length="75" --min_length="5" --beam_search --caption_extension=".txt"
                         "C:/Users/nicol/Documents/Taocah_Training/Images/taocah_training_images"
                         --caption_weights="https://storage.googleapis.com/sfr-vision-language-research/BLIP/models/mode                         l_large_caption.pth"
Traceback (most recent call last):
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\finetune\make_captions.py", line 16, in <module>
    from blip.blip import blip_decoder
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\finetune\blip\blip.py", line 13, in <module>
    from blip.vit import VisionTransformer, interpolate_pos_embed
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\finetune\blip\vit.py", line 16, in <module>
    from timm.models.vision_transformer import _cfg, PatchEmbed
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\venv\Lib\site-packages\timm\__init__.py", line 2, in <module>
    from .models import create_model, list_models, is_model, list_modules, model_entrypoint, \
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\venv\Lib\site-packages\timm\models\__init__.py", line 28, in <module>    from .maxxvit import *
  File "C:\Users\nicol\Documents\AI\khoya\kohya_ss\venv\Lib\site-packages\timm\models\maxxvit.py", line 225, in <module>    @dataclass
     ^^^^^^^^^
  File "C:\Users\nicol\AppData\Local\Programs\Python\Python311\Lib\dataclasses.py", line 1230, in dataclass
    return wrap(cls)
           ^^^^^^^^^
  File "C:\Users\nicol\AppData\Local\Programs\Python\Python311\Lib\dataclasses.py", line 1220, in wrap
    return _process_class(cls, init, repr, eq, order, unsafe_hash,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicol\AppData\Local\Programs\Python\Python311\Lib\dataclasses.py", line 958, in _process_class
    cls_fields.append(_get_field(cls, name, type, kw_only))
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nicol\AppData\Local\Programs\Python\Python311\Lib\dataclasses.py", line 815, in _get_field
    raise ValueError(f'mutable default {type(f.default)} for field '
ValueError: mutable default <class 'timm.models.maxxvit.MaxxVitConvCfg'> for field conv_cfg is not allowed: use default_factory
21:32:22-771800 INFO     ...captioning done
```

I'll attempt to do my own trouble shooting, but I'm curious to see if anyone has encountered this before and if so, how did you fix it?

**Comment by bmaltais** [comment]
I will test it tomorrow...

**Comment by bmaltais** [comment]
Hi, I just tested that to respond to a similar issue and everything work fine on my system under torch 2. Are you using torch 2? You may want to delete the venv folder and re-run .\setup.bat to re-install everything fresh.

**Comment by Starbaby8** [comment]
Reinstalling. One thing I notice before set up is that setup.bat sees that I'm using python 11, but it's not recognizing that I'm using Python 10.10. Would you know how to change that? It could be attributing to the problem. 

![image](https://github.com/bmaltais/kohya_ss/assets/135027507/6ac8e4a8-9984-4457-92cf-e281ef4b67da)

**Comment by Starbaby8** [comment]
Reinstalling. One thing I notice before set up is that setup.bat sees that I'm using python 11, but it's not recognizing that I'm using Python 10.10. Would you know how to change that? It could be attributing to the problem.

![image](https://github.com/bmaltais/kohya_ss/assets/135027507/daac2834-c8ea-441e-b5bb-a4c7565c14e8)

![image](https://github.com/bmaltais/kohya_ss/assets/135027507/16d82c46-1fac-44d3-ba90-458aaeb334fc)

**Comment by bmaltais** [comment]
If you type python --version what does it show?

**Comment by Starbaby8** [comment]
I'm experiencing an unrelated problem, but for some reason python is not being recognized no matter what I do, and I'm unable to proceed with the installation of both Stable or Khoya due to python not being recognizable by my system. I've troubleshooted, tried uninstalling and re installing, changing its path directory in environmental variables and still nothing. I'm still troubleshooting but once I find a solution I'll see if our problem with Blip has been resolved.

![image](https://github.com/bmaltais/kohya_ss/assets/135027507/8bb63f4d-7659-4d6b-8a33-bc8fc432871e)
